src/UKHO.S100PermitService.Common/Models/Permit.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Models
+{
+
+    // NOTE: Generated code may require at least .NET Framework 4.5 or .NET Core/Standard 2.0.
+    /// <remarks/>
+    [ExcludeFromCodeCoverage]
+    [Serializable()]
+    [System.ComponentModel.DesignerCategory("code")]
+    [System.Xml.Serialization.XmlType(AnonymousType = true, Namespace = "http://www.iho.int/s100/se/5.0")]
+    [System.Xml.Serialization.XmlRoot(Namespace = "http://www.iho.int/s100/se/5.0", IsNullable = false)]
+    public partial class Permit
+    {
+        private header headerField;
```

Comment:
Does not conform to code standard, should be _headerField etc.

Reply:
Resolved